src/main/java/com/mongodb/jdbc/MongoConnection.java

```diff
@@ -194,6 +194,29 @@ public void clearWarnings() throws SQLException {
     }
 
     // --------------------------JDBC 2.0-----------------------------
+    @Override
```

Comment:
The 2.0 and 3.0 methods were moved out of the MySQLConnection class into MongoConnection super class because they all just check the direction of cursor then call `createStatement` and `prepareStatement`, so now those are all we need to implement in the subclasses.